content/en/docs/contribute/style/style-guide.md

```diff
@@ -359,6 +361,60 @@ Shortcodes inside include statements will break the build. You must insert them
 {{</* /note */>}}
 ```
 
+
+## Markdown elements
+
+### Line breaks
+Use a single newline to separate block-level content like headings, lists, images, code blocks, and others. The exception is second-level headings, where it should be two newlines. This helps visualize the overall structure of content in a code editor.
```

Comment:
Should we encourage authors to wrap the text of paragraphs? I know that wrapping makes [GitHub suggested changes](https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) more readable.

Reply:
Yes, interesting and a very good point. There are pros and cons to wrapping: https://about.gitlab.com/2016/10/11/wrapping-text/. I will leave it as of now. May be later when we decide about wrapping we can update the wrapping guideline.